src/convertToObject.js

```diff
@@ -3,16 +3,39 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObj = {};
+
+  const stylesArray = sourceString.split(';');
+
+  const prettyStylesArray
+    = stylesArray.map(
+      s => s.replace(/\s\s+/g, ' ')
+        .trim())
+      .filter(s => s);
+
+  prettyStylesArray.map(
+    s => s.replace(/\s+(:)/g, ':'));
```

Comment:
instead of all this you can just `trim()` and `split(':')` 
and filter out blank spaces with `!s.includes('')`